### DISCUSSION TITLE
A FEW PRESTA BACKOFFICE UPGRADES

### DISCUSSION THREAD

**Discussion by BIRDmeble** [PrestaShop/PrestaShop#40142] [Ideas]
### Prerequisites

- [ ] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

I think, no

### Describe the solution you'd like

Have a couple changes to upgrade Presta:

1. More possibility filter products in the catalogue and orders in backoffice, should we able to search by: date, product name, product idnex, customer name/surname, etc. more its better
2. Order and products litings should be load without refreshing all site, should use dynamic content load
3. Should be possibility to massive change any parameters for the products, mean we want change the price for 100 products, and we highlight products and click mass action and select parameter what we want to change for all highlighted products.
4. Backoffice should be faster, its take time to load any site on backoffice

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by marwachelly** [comment]
Hello @BIRDmeble ,

For the point 1 and 3 those features already exist in prestashop as you can see in the screen recording bellow (sort, filters , search , bulk action).

https://user-images.githubusercontent.com/89604350/165300594-498659d4-8ce5-403e-b026-0869e5be0380.mp4

For the point 2 and 4 I think we are aware of this issue and they are the same as #24975. 

Did I miss some think? Could you please check and feedback?

Thanks,

**Comment by BIRDmeble** [comment]
Hi @marwachelly
about filters, sort, search, I hope more like this:

https://www.youtube.com/watch?time_continue=102&v=Nra1I16Abwc&feature=emb_title

Like you see, dynamic loading, many filters.

About mass products edit actions, look at this:

https://www.youtube.com/watch?v=UV-wxN4lqvs

Have another point:
5. Copy products should have options to check some part of product, for example: without gallery, whithout special prices or deliver etc.

**Comment by marwachelly** [comment]
Hello,

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know!

Thanks,
